Scheduler/observing_blocks_maintel/AOS-OR4/BLOCK-276.json

```diff
@@ -0,0 +1,202 @@
+{
+    "name": "BLOCK-276",
+    "program": "BLOCK-276",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "run_command.py",
+            "standard": true,
+            "parameters": {
+                "component": "CCCamera",
+                "cmd": "play",
+                "parameters": {
+                    "playlist": "or4_day1_aos_sweep_cam_dz_v1",
+                    "repeat": true
```

Comment:
Good point. I changed it to false. It was true in BLOCK-202 because we were expecting to keep taking the same images over and over, but not in this case.

Reply:
That is whether the playlist should start again if you get to the end of it.  You do want it as false.